### PR TITLE
[tests-only][full-ci] Add test for updating space name to an empty string

### PR DESCRIPTION
**Pull request by Ashim-Stha**
## Description

Add test for updating space name to an empty string

## Related Issue

https://github.com/owncloud/ocis/issues/11887

### CODE REVIEW

File: tests/acceptance/features/apiSpaces/changeSpaces.feature [hunk 1]
```diff
@@ -597,4 +597,9 @@ Feature: Change data of space
     Examples:
       | role                          |
       | Space Editor Without Versions |
-      | Space Editor                  |
\ No newline at end of file
+      | Space Editor                  |
+
+  @issue-11887
+  Scenario: user with space manager role tries to change the name of a space to an empty string via the Graph API
```
Comment (saw-jan) [inline tests/acceptance/features/apiSpaces/changeSpaces.feature]:
```suggestion
  Scenario: space manager tries to change the name of a space to an empty string
```
